cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/BulkSparkConf.java

```diff
@@ -203,7 +203,7 @@ else if (!batchSizeIsZero && sstableBatchSize != DEFAULT_BATCH_SIZE_IN_ROWS)
     /*
      * This method Will throw if the SSL configuration is incorrect (PATH provided w/o password, for example)
      */
-    protected void validateSslConfiguration()
+    public void validateSslConfiguration()
```

Comment:
Can you enrich the javadoc? It is now a public method. I think it should cover the possible exceptions and all the scenarios when it could throw.

Reply:
Added detailed description of possible failure scenarios and corresponding exception types.